tutorials/assets_pipeline/importing_translations.rst

```diff
@@ -6,10 +6,9 @@ Importing translations
 Games and internationalization
 ------------------------------
 
-The world is full of different markets and cultures and, to maximize
-profits™, nowadays games are released in several languages. To solve
-this, internationalized text must be supported in any modern game
-engine.
+The world is full of different languages and cultures, so nowadays games
+are released in several languages. To solve this, internationalized text
+must be supported in any modern game engine.
```

Comment:
I would change this to "To support this"
We've not solving the problem of the world having different languages, we're not undoing babylon ;)

Reply:
Or some change to that effect that actually makes the sentence flow, you know what I mean :)

Reply:
Yeah, that makes sense! I'll update the PR...